robottelo/ui/computeresource.py

```diff
@@ -191,6 +193,20 @@ def set_values(self, **kwargs):
         if template is not None:
             self.set_value(template_key, template)
             del kwargs[template_key]
+        # when setting memory value it does not fire the change event,
+        # that do the necessary validation and update the memory hidden field,
+        # without this event fired the memory value cannot be saved,
+        memory_key = 'memory'
```

Comment:
don't like that `set_value` method at all as too much of redundant code, other than that, looks good